tasks/auditd.yml

```diff
@@ -0,0 +1,14 @@
+---
+
+- name: install auditd package | package-08
+  package:
+    name: '{{ auditd_package }}'
+    state: 'present'
+
+- name: configure auditd | package-08
+  template:
+    src: 'auditd.conf.j2'
```

Comment:
In larger Ansible projects it quickly pays of when you recreate the directory structure also for template source files. E.g.:

```YAML
    src: '/etc/audit/auditd.conf.j2'
```

Thats what we do in DebOps.

Other than that I can not review this PR any further right now or approve it because I don’t know all the auditd configuration stuff added by this PR and if they are sensible defaults.

Reply:
Fixed the sources.

As to the defaults of audit: these are the debian defaults, no special hardening used.